styles/variables-silicon.less

```diff
@@ -176,39 +176,51 @@
 
 // Input
 // ---------------------------------------
-@agate-input-border-width: 3px;
+@agate-input-decorator-border-bottom-width: 3px;
 @agate-input-decorator-border-width: 0;
 @agate-input-decorator-focus-border-width: 0;
+@agate-input-decorator-has-value-opacity: 0.7;
 @agate-input-decorator-height: 60px;
+@agate-input-decorator-opacity: 0.4;
+@agate-input-disabled-opacity: 0.3;
```

Comment:
Please check the names and value again.

Reply:
I changed the names and regarding the values, they are different in some cases because there is a combination of opacities between the input decorator and the elements inside, but I checked the border, icons and text with a color picker and the value is as close as possible to the values in GUI